server/schemas/field.graphql

```diff
@@ -21,6 +21,7 @@ type SchemaField {
   typeProperty: SchemaFieldTypeProperty
   key: String!
   title: String!
+  order: Int!
```

Comment:
```suggestion
  order: Int
```

If there is no order, it should be automatically decided

Reply:
it's already being automatically decided. thank you